src/core/json/jsonpath_test.cc

```diff
@@ -30,15 +30,25 @@ class TestDriver : public Driver {
   }
 };
 
-inline JsonType ValidJson(string_view str) {
+template <typename JSON> JSON ValidJson(string_view str);
```

Comment:
there are no functional changes in this file, only syntax sugar according to https://google.github.io/googletest/advanced.html#typed-tests